jams/eval.py

```diff
@@ -54,10 +54,7 @@ def validate_annotation(ann, namespace):
         If `ann` fails schema validation
     '''
 
-    if not ann.search(namespace=namespace):
-        raise NamespaceError('Expected namespace="{:s}", found "{:s}"'
-                             .format(namespace, ann.namespace))
-
+    ann = convert(ann, namespace)
```

Comment:
@bmcfee am I right in understanding that you're validating the jams by converting it to the desired namespace and then checking if it validates against this namespace?

If so, doesn't this mean you're not only doing validation, but also conversion, under the hood?